### ISSUE TITLE
Selenium-side-runner cannot run tests in Firefox

### ISSUE THREAD

**Issue by sergiybuch** [SeleniumHQ/selenium-ide#490]
## 🐛 Bug Report

Unable to run Selenium IDE tests in Firefox using SIDE Runner.
When I open Selenium IDE Extension in Firefox and open my project, then it runs tests without any issues, but as soon as I change chrome -> firefox in capabilities I am getting errors  
![image](https://user-images.githubusercontent.com/28896994/48795318-79741600-ecca-11e8-9790-bc9d817dcc2d.png)

Adding [project.side and .side.yml](https://drive.google.com/open?id=1DVDbemddnp_IEqgJ2Yz7gTaw2Q1uJtI2)

## Environment

OS: 
Selenium IDE Version:  3.4.2
Selenium SIDE Runner Version:  <3.4.2
-->
Node version: 8.9.4
Only required if using selenium-side-runner
-->
Browser: Firefox  
Browser Version: 63.0.3 32 bit

**Comment by tourdedave** [comment]
This is a duplicate of #329.

tl;dr -- The Selenium bindings that the runner uses has unimplemented commands (e.g., commands that use Selenium Actions like `mouse down at`, `mouse move at`, etc.).

We're hoping to have FF support back in the runner by the time we ship 3.5.0.
